src/test/java/com/rultor/agents/daemons/EndsDaemonITCase.java

```diff
@@ -103,6 +105,38 @@ public void readsExitCodeCorrectly() throws IOException {
     }
 
     /**
+     * EndsDaemon can deprecate default image.
+     * @throws IOException In case of error
+     * @todo #754:30min Implement a deprecation message at the start
+     *  of the process if the project is using the default image
+     *  'yegor256/rultor'. Fix the issue using the xpath available in
+     *  com.rultor.agents.daemons.EndsDaemon
```

Comment:
@sebing can you also mention #1018 at the end of description? It contains very useful data